### PR TITLE
chore: several changes to reflect the mono repo

### LINKED ISSUE #545: amend readme.md
- [ ] npx prisma studio does not run any more: repair and document how to run
- [ ] remove ```npm ...``` parts and replace with ```pnpm ....``` and double check if they work
- [ ]

### PR DESCRIPTION
**Pull request by moncapitaine**
closes #545

### CODE REVIEW

**Comment by moncapitaine** [comment]
closes #545

File: README.md [hunk 1]
```diff
@@ -14,25 +16,48 @@ Windows users may have trouble with the default installation folder of pnpm:
...
+
+- **packages/pulumi**  
+  the infrastructure as code package build with pulumi
+
+- **packages/ui**  
+  re-usable UI components
+
+- **packages/tsconfig**  
+  central place to store all tsconfigs for all packages that use tsc
+
```
Comment (PascalHelbig) [inline README.md:38]: What about the `validations` and `eslint-config-custom` folders?

File: README.md [hunk 2]
```diff
@@ -14,25 +16,48 @@ Windows users may have trouble with the default installation folder of pnpm:
...
 ```bash
-docker-compose up
+docker-compose up -d
 ```
 
 This app is developed using prisma.io. You can start the prisma studio to view/modify data by using:
 
 ```bash
+cd packages/backend
 npx prisma studio
```
Comment (PascalHelbig) [inline README.md:61]:
```suggestion
pnpx prisma studio
```

File: README.md [hunk 3]
```diff
@@ -14,34 +16,57 @@ Windows users may have trouble with the default installation folder of pnpm:
...
-docker-compose up
+docker-compose up -d
 ```
 
 This app is developed using prisma.io. You can start the prisma studio to view/modify data by using:
 
 ```bash
-npx prisma studio
+cd packages/backend
+pnpx prisma studio
```
Comment (YohannesTesfay) [inline README.md:61]: I think this won't work with our single `.env` file in the root folder.

What about if we provide the `prisma.schema` location in the root `package.json`? I tried it and it worked for me.

Comment (moncapitaine) [reply README.md:67]: I added a turbo script
```bash
pnpm prisma-studio
```

This can be executed without having to go into the backend directory first.

**Review by Slash52 — APPROVED** [review]

**Pull request merged**